source/agora/consensus/data/Transaction.d

```diff
@@ -249,6 +306,12 @@ public struct Input
         dg(this.utxo[]);
         hashPart(this.unlock_age, dg);
     }
+
+    /// Support for sorting
+    public int opCmp ( const typeof(this) rhs ) const nothrow @safe @nogc
```

Comment:
```suggestion
    public int opCmp ( const typeof(this) rhs ) const @safe nothrow @nogc
```
I know "make @safe the first attribute" is our policy.

Reply:
Thanks I updated